mozilla-release/browser/base/content/tabbrowser.xml

```diff
@@ -2714,63 +2714,12 @@
 
               this.tabContainer.updateVisibility();
 
-<<<<<<< HEAD
-            // If we open a new tab with the newtab URL in the default
```

Comment:
so AFT does not work?

Reply:
It's work now. This code moved a little bit down